components/dashboard/src/contexts/FeatureFlagContext.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import React, { createContext, useState } from "react";
+
+const FeatureFlagContext = createContext<{
+    isBlockedRepositoriesUIEnabled: boolean;
+    setIsBlockedRepositoriesUIEnabled: React.Dispatch<boolean>;
+}>({
+    isBlockedRepositoriesUIEnabled: false,
+    setIsBlockedRepositoriesUIEnabled: () => {},
+});
+
+const FeatureFlagContextProvider: React.FC = ({ children }) => {
+    const [isBlockedRepositoriesUIEnabled, setIsBlockedRepositoriesUIEnabled] = useState(false);
```

Comment:
Nice! Was thinking about whether this make sense as well.

:bulb: I it possible to `useUserContext()` here, and the the values with an `useEffect` in here? :thinking: 
(this is clearly something for a follow up!)